pkg/drivers/kic/oci/network_create.go

```diff
@@ -36,6 +37,9 @@ const firstSubnetAddr = "192.168.49.0"
 // big enough for a cluster of 254 nodes
 const defaultSubnetMask = 24
 
+// will be used if docker bridge config doesn't exist related issue #9528
+const defaultNetworkMTU = 1500
```

Comment:
How do you feel about `defaultMTU` as the name? I feel that it would be just as informative, but feel less cluttery.

Reply:
agree, lgtm otherwise